workflow/load/src/main/java/com/datastax/oss/dsbulk/workflow/load/LoadWorkflow.java

```diff
@@ -234,6 +237,7 @@ public boolean execute() {
     statements
         .transform(this::executeStatements)
         .transform(queryWarningsHandler)
+        .transform(failedWritesMonitor)
```

Comment:
This was the root cause of the bug: the monitor that increments the `records/failed` metric wasn't being invoked here, and so failed writes weren't being accounted for.